src/pieces/commandManager.ts

```diff
@@ -94,7 +94,12 @@ async function loadCommands(bot: Client) {
 	let permsUpdated = 0;
 	console.log('Checking for updated permissions...');
 	await Promise.all(commands.cache.map(async command => {
-		const curPerms = await command.permissions.fetch({ command: command.id });
+		let curPerms: ApplicationCommandPermissionData[];
```

Comment:
This shouldn't need to be in this PR, seems like you didn't rebase after the most recent commit on the `slashCommands` branch. Fix this by doing a 
```
$ git checkout slashCommands
$ git pull
$ git checkout bladeSlashification
$ git rebase slashCommands
```
If you run into trouble, let me know

Reply:
All done. Do I push to BladeSlashification after the rebase?